WickedEngine/wiRenderer.cpp

```diff
@@ -2385,6 +2385,36 @@ ForwardEntityMaskCB ForwardEntityCullingCPU(const Visibility& vis, const AABB& b
 	return cb;
 }
 
+void Workaround(const int bug , CommandList cmd)
+{
+	if (device->GetBackend() == BACKEND_DX12)
+	{
+		if (bug == 1)
+		{
+			//PE: Strange DX12 bug, we must change the pso/pipeline state, just one time.
```

Comment:
It would be good to have a `device->EventBegin()` and `device->EventEnd()` surrounding workaround gfx commands, so that workaround is obvious when gfx debugging.